clients/cli/src/unwrap.rs

```diff
@@ -206,12 +187,14 @@ pub async fn command_unwrap(
         config.rpc_client.get_latest_blockhash().await?
     };
 
+    let payer = config.fee_payer()?;
+
     // Payer will always be a signer
-    let mut signers: Vec<&dyn Signer> = vec![payer.as_ref()];
+    let mut signers: Vec<Arc<dyn Signer>> = vec![payer.clone()];
```

Comment:
nit: Was this change necessary? Since the signers all outlive this vec, `Vec<&dyn Signer>` should be fine, unless I'm missing something